content/html/elements/video/prose.md

```diff
@@ -1,9 +1,7 @@
-<!-- <short-description> -->
+<!-- short-description -->
```

Comment:
Yes... I'm not sure now why we opted for this plan rather than just using headings. Also I'm not happy that prose.md uses comments while attributes use headings.

So unless someone can remember why we decided to do this, I think you are right.

Again these are things we can tweak as we go along though.

Reply:
I'm going to keep this for now but filed https://github.com/mdn/stumptown-experiment/issues/19.

Reply:
This comment doesn't signify that I've done any kind of thorough review 😃, but I was skimming this PR while waiting for some tests to finish and noticed this thread. I can't remember the details either, but I'm also not happy with the fact that `prose.md` uses hidden comments to enforce structure while the attributes use headings. It's probably best to choose some specific headings for signifying/enforcing structure, and get rid of the hidden comments. So, for what it's worth, just wanted to say that I agree with you both in where you're heading here! 😄